src/Eloquent/Model.php

```diff
@@ -195,7 +195,7 @@ public function setAttribute($key, $value)
 
             $value = $builder->convertKey($value);
         } // Support keys in dot notation.
-        elseif (Str::contains($key, '.')) {
+        elseif (str_contains($key, '.')) {
```

Comment:
Where does this package require PHP 8+? I don't see it in `composer.json` so I reckon it's an indirect dependency. I think it'd make sense to explicitly require it if we're going to be utilizing PHP 8+ functionality directly.

Hhappy to defer to a subsequent PHPORM ticket.

Reply:
It requires `laravel/framework: ^10.0` which [requires PHP 8.1+](https://github.com/laravel/framework/blob/10.x/composer.json#L18).